mkdocs-project-dir/docs/Account_Services.md

```diff
@@ -1,68 +1,137 @@
+## Accounts for Visitors
 
-## Accounts for visitors
+UCL visitors are welcome to apply for accounts on Research Computing Services.
+Please note that:
 
-UCL visitors are welcome to apply for accounts on Research Computing services. Please note that:
+1. Applicants must have a central UCL account. UCL Visitor accounts can be
+   arranged by your Departmental Administrator, so you should speak to them
+   first.
+2. Account applications should specify the UCL grant under which the work is
+   being carried out, if possible, as well as an associated UCL Group or
+   Researcher.
+3. Account applications may not be submitted on behalf of another, except to
+   cover accessibility requirements, as the account application process includes
+   agreeing to relevant legal terms and conditions.
 
-1. Applicants must have a central UCL account. UCL Visitor accounts can be arranged by your Departmental Administrator, so you should speak to them first.
-2. Account applications should specify the UCL grant under which the work is being carried out, if possible, as well as an associated UCL group or researcher.
-3. Account applications may not be submitted on behalf of another except to cover accessibility requirements, as the account application process includes agreeing to relevant legal terms and conditions.
+## Accounts for Honorary Staff
 
-## Accounts for honorary staff
+UCL Staff Members may nominate Honorary members (named individuals) to be
+provided with access to Research Computing services where this is beneficial to
+UCL's research interests.
 
-UCL Staff Members may nominate Honorary members (named individuals) to be provided with access to Research Computing services where this is beneficial to UCL's research interests.
+Nomination should be made via the
+[CRAG](http://www.ucl.ac.uk/isd/about/governance/research-it/crag), explaining
+the benefit arising to UCL.
 
-Nomination should be made via the [CRAG](http://www.ucl.ac.uk/isd/about/governance/research-it/crag), explaining the benefit arising to UCL.
+<!--- TODO: (them): CRAG seems to have no page of its own:             --->
+<!--- it redirects me to a head UCL page, which wants me to login anew --->
+<!--- CRAG is mentioned several times below                            --->
 
-Proposals will be reviewed by the [CRAG](http://www.ucl.ac.uk/isd/about/governance/research-it/crag) at their monthly meeting and on a case by case basis.
+Proposals will be reviewed by the
+[CRAG](http://www.ucl.ac.uk/isd/about/governance/research-it/crag) at their
+monthly meeting and on a case by case basis.
 
-All accounts thus provided are subject to all other 'standard' T&C's relating to their use of Research Computing services. 
+All accounts thus provided are subject to all other 'standard' T&C's relating to
+their use of Research Computing Services.
 
-## Charges for use of Research Computing services
+## Charges for use of Research Computing Services
```

Comment:
No, should be lower-case services because it is referring to our services generically. While 'Research Computing Services' would be a named service.